flutter_bird_app/lib/controller/authentication_service.dart

```diff
@@ -349,6 +352,10 @@ class AuthenticationServiceImpl implements AuthenticationService {
   Uri _convertToWcUri({
     required String appLink,
     required String wcUri,
-  }) =>
-      Uri.parse('$appLink/wc?uri=${Uri.encodeComponent(wcUri)}');
+  }) {
+    if (appLink[appLink.length - 1] == '/') {
+      appLink = appLink.substring(0, appLink.length - 1);
+    }
```

Comment:
> Fix _convertToWcUri - This function returned invalid uri ('metamask:///wc?url=xxxx', three slashes is invalid).

I think this is the fix you said, but would you mention it in the comments as well?